### PR TITLE
Feat/#95 applicant dash feedback

### PR DESCRIPTION
**Pull request by ciaranschutte**
- use `react-grid-system` like other pages
- adjust spacing between items
- minor text changes

### CODE REVIEW

File: components/pages/Applications/Dashboard/Applications/index.tsx [hunk 1]
```diff
@@ -4,28 +4,30 @@ import { styled, UikitTheme } from '@icgc-argo/uikit/index';
...
+    <div
+      css={css`
+        flex: 1 0 auto;
+      `}
+    >
+      <div
+        css={css`
+          display: grid;
+          grid-gap: 24px;
+          grid-template-columns: repeat(auto-fit, minmax(280px, 1fr));
```
Comment (ciaranschutte) [inline components/pages/Applications/Dashboard/Applications/index.tsx:24]: change to look slightly better when diminishing screen size (page not fully responsive at all yet)

File: components/pages/Applications/Dashboard/Applications/index.tsx [hunk 2]
```diff
@@ -4,28 +4,30 @@ import { styled, UikitTheme } from '@icgc-argo/uikit/index';
 import React from 'react';
 import InProgress from './InProgress';
 import StartApplication from './Start';
+import { css } from '@emotion/core';
 
 type ApplicationsProps = {
   inProgressApplications: any;
 };
 
-const TwoColGrid = styled('div')`
```
Comment (ciaranschutte) [inline components/pages/Applications/Dashboard/Applications/index.tsx:12]: remove abysmal naming and make it inline css

File: components/pages/Applications/Dashboard/index.tsx [hunk 3]
```diff
@@ -1,12 +1,50 @@
...
 import Applications from './Applications';
-import DashboardHeader from './Header';
+import AccessBox from './AccessBox';
 
 const Dashboard = () => {
   return (
     <>
-      <DashboardHeader />
-      <Applications inProgressApplications={[]} />
+      <PageHeader>My Applications</PageHeader>
```
Comment (ciaranschutte) [inline components/pages/Applications/Dashboard/index.tsx:11]: move some singular files into main index. easier to reason about grid system

File: components/pages/Applications/Dashboard/index.tsx [hunk 4]
```diff
@@ -1,12 +1,50 @@
...
+      <Container
+        css={css`
+          margin-top: 24px;
+        `}
+      >
+        <Row
```
Comment (ciaranschutte) [inline components/pages/Applications/Dashboard/index.tsx:17]: a key difference in this page is that a `card` style component isn't used for the entire section therefore we need to remove the gutter so it aligns with the page header

File: components/pages/Applications/Dashboard/index.tsx (hunk 3 above)
Comment (samrichca) [reply components/pages/Applications/Dashboard/index.tsx:11]: hurray this PageHeader is what i was looking for!

File: components/pages/Applications/Dashboard/Applications/index.tsx [hunk 5]
```diff
@@ -4,28 +4,30 @@ import { styled, UikitTheme } from '@icgc-argo/uikit/index';
 import React from 'react';
 import InProgress from './InProgress';
 import StartApplication from './Start';
+import { css } from '@emotion/core';
```
Comment (samrichca) [inline components/pages/Applications/Dashboard/Applications/index.tsx:7]: good, good

**Review by samrichca — APPROVED** [review]
layout implementation looks good! 🎨

**Pull request merged**